### PR TITLE
Enhancements for the rd6006.

### PR DESCRIPTION
**Pull request by phsdv**
Added function to read memories. A __main__ for easy testing, corrected  shift 16 iso 8 bits as 16 bit registers. Added also: battery voltage, ocpovp, CVCC check and added simple time and date setters.

### CODE REVIEW

File: rd6006/rd6006.py [hunk 1]
```diff
@@ -131,10 +152,59 @@ def enable(self):
     @enable.setter
     def enable(self, value):
         self._write_register(18, int(value))
+    
+    @property
+    def ocpovp(self):
+        m = self._read_register(16)
```
Comment (Baldanos) [inline rd6006/rd6006.py:158]: You should directly return the value here instead of setting a variable (more consistency)

File: rd6006/rd6006.py [hunk 2]
```diff
@@ -131,10 +152,59 @@ def enable(self):
...
+
+    @property
+    def date(self):
+        regs = self._read_registers(48,3)
+        y = regs[0]
+        m = regs[1]
+        d = regs[2]
+        return(y,m,d)   
+    @date.setter
+    def date(self, value):
```
Comment (Baldanos) [inline rd6006/rd6006.py:180]: Add a docstring to explain arguments (not obvious from within the shell)

File: rd6006/rd6006.py [hunk 3]
```diff
@@ -131,10 +152,59 @@ def enable(self):
...
+
+    @property
+    def time(self):
+        regs = self._read_registers(51, 3)
+        h = regs[0]
+        m = regs[1]
+        s = regs[2]
+        return(h, m, s)   
+    @time.setter
+    def time(self, value):
```
Comment (Baldanos) [inline rd6006/rd6006.py:194]: Add a docstring to explain arguments (not obvious from within the shell)

File: rd6006/rd6006.py (hunk 1 above)
Comment (phsdv) [reply rd6006/rd6006.py]: yes, agree

**Pull request merged**